packages/extension/src/components/pages/Login/Login.tsx

```diff
@@ -130,6 +138,21 @@ export const Login: FC = () => {
     loadRememberSession();
   }, []);
 
+  useEffect(() => {
+    const loadTimerData = async () => {
+      const storedTimerData = await loadFromChromeLocalStorage('disabledLoginTimer');
+      if (storedTimerData !== null) {
+        const expirationDate = storedTimerData;
+        if (expirationDate > Date.now()) {
+          setDisableLogin(true);
```

Comment:
```suggestion
          setDisableLogin(true);
          setPasswordError('Too many attempts, please try again later');
```

So when we open again the extension, we see the error message. otherwise, having only the greyed button may look like a bug IMO

Reply:
Thanks for the review, I did not think of that. 
The improvement is done in commit : https://github.com/GemWallet/gemwallet-extension/pull/275/commits/dbfb73c08bb9b9e4e1c2aba5ee4b0b02a4f44643